src/openrct2/util/Util.cpp

```diff
@@ -188,10 +188,10 @@ sint32 bitscanforward(sint32 source)
 #endif
 
 #ifdef OPENRCT2_X86
-static bool cpuid_x86(uint32 * cpuid_outdata, sint32 eax)
+static bool cpuid_x86(uint32 * cpuid_outdata, sint32 eax, sint32 ecx = 0)
 {
 #if defined(OpenRCT2_CPUID_GNUC_X86)
-    int ret = __get_cpuid(eax, &cpuid_outdata[0], &cpuid_outdata[1], &cpuid_outdata[2], &cpuid_outdata[3]);
+    int ret = __get_cpuid_count(eax, ecx, &cpuid_outdata[0], &cpuid_outdata[1], &cpuid_outdata[2], &cpuid_outdata[3]);
```

Comment:
Ugh, they added a new call that allows specifying subleaf in recent GCC version, but removed the legacy behaviour which specified subleafs for the user.

I think I'll have to change the avx2 detection to use GCC's builtin.